Makefile

```diff
@@ -1,3 +1,8 @@
 .PHONY: init
 init:
 	cd virtme_ng_init && cargo install --path . --root ../virtme/guest
+
+# see README.md '* Install from source'
+install: install_from_source
+install_from_source:
+	BUILD_VIRTME_NG_INIT=1 pip3 install --verbose -r requirements.txt .
```

Comment:
Can we add something like the following?
```
diff --git a/Makefile b/Makefile
index 23f76a3..6925d8c 100644
--- a/Makefile
+++ b/Makefile
@@ -1,3 +1,5 @@
+INSTALL_ARGS :=
+
 .PHONY: init
 init:
 	cd virtme_ng_init && cargo install --path . --root ../virtme/guest
@@ -5,4 +7,4 @@ init:
 # see README.md '* Install from source'
 install: install_from_source
 install_from_source:
-	BUILD_VIRTME_NG_INIT=1 pip3 install --verbose -r requirements.txt .
+	BUILD_VIRTME_NG_INIT=1 pip3 install --verbose -r requirements.txt $(INSTALL_ARGS) .
```

In this way we can pass extra install arguments (such as `--break-system-packages` that is required in Ubuntu).

Reply:
that makes sense.
Id seen that example and meant to ask if it belonged in the Makefile

if you want to modify my patch and commit it - great.
or I could revise it.

Reply:
I got it